lms/templates/video.html

```diff
@@ -17,6 +17,8 @@ <h3 class="hd hd-2">${display_name}</h3>
     data-bumper-metadata='${bumper_metadata}'
     data-autoadvance-enabled="${autoadvance_enabled}"
     data-poster='${poster}'
+    data-block-id='${block_id}'
```

Comment:
This data exists in the next level up, in the div that represents the abstract xblock, but adding it here simplifies testing and generally makes the video player more encapsulated